recipe/meta.yaml

```diff
@@ -17,8 +17,9 @@ source:
     - lcurl-pkg-config.patch
 
 build:
-  number: 6
```

Comment:
Shouldn't we bump build-number here?

Reply:
Uhm, it's bumped to 7 two lines further below?